open_discussions_api/users/client.py

```diff
@@ -56,6 +57,7 @@ def create(self, email=None, profile=None):
 
         payload = {
             'profile': profile or {},
+            'email_optin': email_optin,
```

Comment:
`email_optin` needs to be passed in `profile`, not the root payload. See https://github.com/mitodl/open-discussions/blob/master/profiles/views_test.py#L36